documentation/contributing/future_projects/index.markdown

```diff
@@ -34,26 +52,19 @@ MoveIt is not a perception library, but does come with support for using octomap
 
```

Comment:
Let's remove the perception one, we had a GSoC student do a decent job here @mlautman ?